jdaviz/conftest.py

```diff
@@ -290,6 +292,12 @@ def mos_image():
     return CCDData(data, wcs=wcs, unit='Jy', meta=header)
 
 
+@pytest.fixture
+@pytest.mark.skipif(not HAS_ROMAN_DATAMODELS, reason="roman_datamodels is not installed")
```

Comment:
Putting a skipif in a fixture is too confusing. If you want a test to skip, it is more explicit to put skipif on the test function.

If  `create_wfi_image_model` needs Roman datamodels, then maybe have this return None if that package isn't installed with a if-else?

Reply:
Alternately, you can move all the stuff that needs Roman data models into a special test module and use `pytest.importorskip`.

Reply:
As of 8a12ef8, we do both. There's a `roman_datamodels`-specific test in `jdaviz/configs/imviz/tests/test_parser_roman.py` that uses `importorskip`, and now the fixture returns None unless `roman_datamodels` is available.